.github/workflows/CI-bump-build-number.yml

```diff
@@ -0,0 +1,25 @@
+name: CI - Update Build Number
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+jobs:
+  upadate-build-number:
+    name: Update Build Number
+    runs-on: macos-12
```

Comment:
Haven't we updated to macOS 13 build target yet?

Reply:
There's no macOS 13 GitHub Actions runner out yet.